### ISSUE TITLE
Payara 5.184: Nonpersistent timer creation fails at startup

### ISSUE THREAD

**Issue by pselin123** [payara/Payara#3491]
# Description #
----------
While trying to create timer, a NullpointeException is thrown and deployment is failed.
In previous Payara version the timer was created and the timeout method was called as expected.

The exception
```
Caused by: java.lang.NullPointerException
	at com.sun.ejb.containers.EJBTimerServiceWrapper.createTimerInternal(EJBTimerServiceWrapper.java:337)
	at com.sun.ejb.containers.EJBTimerServiceWrapper.createIntervalTimer(EJBTimerServiceWrapper.java:164)
	at com.example.MyStartupClass.startStatupTimer(MyStartupClass.java:35)
```

Might be related to [Recent timer service changes](https://github.com/payara/Payara/commit/cd52a194d9daa0113ef283cdbaabfa0aec96e265)

### Samples

Problematic code:
```
@Startup
@Singleton
public class MyStartupClass {

    @Resource
    private TimerService timeService;

    private Timer timer;

    @Inject
    public MyStartupClass() {
    }

    @PostConstruct
    public void OnStartup() {
        TimerConfig config = new TimerConfig();
        config.setPersistent(false);
        timer = timeService.createIntervalTimer(10000, 2000, config);
    }


    @Timeout
    private synchronized void OnTimer() {
    }

    @PreDestroy
    public void OnStop() {
        if(timer != null) {
            timer.cancel();
            timer = null;
        }
    }
}
```

## Environment ##

- **Payara Version**: 5.184
- **Edition**:  Full
- **JDK Version**:  8  u191
- **Operating System**: Linux

**Comment by smillidge** [comment]
Can you create a simple maven project hosted on GitHub as a test case please?

**Comment by chehl** [comment]
The same issue also exists in Payara 4.1.2.184.

Stack trace:
```
Caused by: java.lang.NullPointerException
		at com.sun.ejb.containers.EJBTimerServiceWrapper.createTimerInternal(EJBTimerServiceWrapper.java:337)
		at com.sun.ejb.containers.EJBTimerServiceWrapper.createIntervalTimer(EJBTimerServiceWrapper.java:164)
```

The "NonPersistentEJBTimerService" is null. It has been introduced in the following commit:

https://github.com/payara/Payara/commit/cd52a194d9daa0113ef283cdbaabfa0aec96e265#diff-f6ff98c09ac3a1a5b22ae6867cecd6dd

**My environment:**
- Payara Version: 4.1.2.184
- Java 8
- Operating System: Linux

**Comment by smillidge** [comment]
Fixed by #3502
